### PR TITLE
Version6 cleanup merge by mistake

### PR DESCRIPTION
**Pull request by juliodialpad**
## Description
Merged version6 by mistake, reverted all the changes and cherry-picked all the necessary commits to get it to it stable version,

## Pull Request Checklist

 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Provide a description what is being changed, extended, or removed.
 - [x] Request a review from Drew Chandler, Joshua Hynes, or Ted Goas.

### CODE REVIEW

File: docs/utilities/typography/font-family.html [hunk 1]
```diff
@@ -95,3 +95,40 @@
     </footer>
   </aside>
 </section>
+
+<section class="d-stack16">
```
Comment (dbecherdialpad) [inline docs/utilities/typography/font-family.html:99]: does this branch need a rebase?

**Review by dbecherdialpad — COMMENTED** [review]
not sure if I had any outstanding comments but @dialpad-drew might have

File: docs/utilities/typography/font-family.html (hunk 1 above)
Comment (dialpad-drew) [reply docs/utilities/typography/font-family.html:99]: I don't think I have any other than what you're asking here, @dbecherdialpad

Comment (juliodialpad) [reply docs/utilities/typography/font-family.html:99]: I don't think so, with just a merge should be fine

**Review by dialpad-drew — APPROVED** [review]

**Comment by dbecherdialpad** [comment]
Sorry, this was supposed to be the Toast PR? I only saw font-family changes which I thought were already merged. Whats the status of DT-95?

**Comment by juliodialpad** [comment]
> Sorry, this was supposed to be the Toast PR? I only saw font-family changes which I thought were already merged. Whats the status of [DT-95](https://switchcomm.atlassian.net/browse/DT-95)?

this PR included all of those changes too, I'm checking and the only one that seems to be missed is the **positon: fixed** that seems to be wrong

**Comment by dialpad-drew** [comment]
> > Sorry, this was supposed to be the Toast PR? I only saw font-family changes which I thought were already merged. Whats the status of [DT-95](https://switchcomm.atlassian.net/browse/DT-95)?
> 
> this PR included all of those changes too, I'm checking and the only one that seems to be missed is the **positon: fixed** that seems to be wrong

Shit, you're right. I totally missed that. 😭

**Pull request merged**